src/background/toolbarButton.ts

```diff
@@ -30,7 +30,7 @@ import {
 import { openNewPontoonTab } from '@commons/utils';
 import { colors } from '@frontend/commons/const';
 
-import { refreshData } from './RemotePontoon';
+import { refreshData, updateNotificationsData } from './RemotePontoon';
```

Comment:
```suggestion
import { refreshData } from './RemotePontoon';
```

This import is not used here.